Dockerfile

```diff
@@ -3,6 +3,20 @@ FROM bitwalker/alpine-elixir:1.4.2
 MAINTAINER PRX <sysadmin@prx.org>
 LABEL org.prx.app="yes"
 
+# install git, aws-cli
+RUN apk update && apk --update add \
+    ca-certificates \
+    git
+
+RUN apk --no-cache update && \
+    apk --no-cache add python py-pip py-setuptools ca-certificates groff less && \
+    pip --no-cache-dir install awscli && \
```

Comment:
great. I honesty didn't check for an apk install, as the blogs/search I did turned up this method, but it makes sense. I'll try that `aws-cli`

and I didn't know that `--no-cache` option, that is sweet, makes me want to update all the projects and Dockerfiles.